### PR TITLE
Calendar: Add selected date range into aria-label

### PR DESCRIPTION
**Pull request by marker-dao**

### CODE REVIEW

File: packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/calendar.markup.tests.js [hunk 1]
```diff
@@ -419,27 +422,184 @@ QUnit.module('Aria accessibility', {
...
+                    value: date,
+                });
+
+                const formattedDate = dateLocalization.format(date, ARIA_LABEL_DATE_FORMAT);
+                const expectedAriaLabel = `Calendar. Selected date is ${formattedDate}`;
+
+                checkTableAttribute(assert, this.$element, 'aria-label', expectedAriaLabel);
+            });
+
+            QUnit.test(`table aria-label should contain selected date when selectionMode=multiple, zoomLevel=${zoomLevel}`, function(assert) {
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/calendar.markup.tests.js]: seems like title is incorrect. 
we assert that title contains only "Calendar"

File: packages/devextreme/js/localization/messages/ar.json [hunk 2]
```diff
@@ -306,6 +306,8 @@
         "dxCalendar-captionYearLabel": "Year selection",
         "dxCalendar-captionDecadeLabel": "Decade selection",
         "dxCalendar-captionCenturyLabel": "Century selection",
+        "dxCalendar-selectedDate": "Selected date is {0}",
+        "dxCalendar-selectedDateRange": "Selected date range from {0} to {1}",
```
Comment (ksercs) [inline packages/devextreme/js/localization/messages/ar.json]: have u validated this variables?

File: packages/devextreme/js/localization/messages/ar.json [hunk 3]
```diff
@@ -306,6 +306,8 @@
         "dxCalendar-captionYearLabel": "Year selection",
         "dxCalendar-captionDecadeLabel": "Decade selection",
         "dxCalendar-captionCenturyLabel": "Century selection",
+        "dxCalendar-selectedDate": "Selected date is {0}",
```
Comment (vladaskorohodova) [inline packages/devextreme/js/localization/messages/ar.json]:
```suggestion
        "dxCalendar-selectedDate": "The selected date is {0}",
```

File: packages/devextreme/js/localization/messages/ar.json (hunk 2 above)
Comment (vladaskorohodova) [inline packages/devextreme/js/localization/messages/ar.json]:
```suggestion
        "dxCalendar-selectedDateRange": "The selected date range is from {0} to {1}",
```

File: packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/calendar.markup.tests.js (hunk 1 above)
Comment (marker-dao) [reply packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/calendar.markup.tests.js]: [c525f2caa6f2747a171668e46989ae9a26af2d31]

File: packages/devextreme/js/localization/messages/ar.json (hunk 3 above)
Comment (marker-dao) [reply packages/devextreme/js/localization/messages/ar.json]: [c525f2caa6f2747a171668e46989ae9a26af2d31]

File: packages/devextreme/js/localization/messages/ar.json (hunk 2 above)
Comment (marker-dao) [reply packages/devextreme/js/localization/messages/ar.json]: [c525f2caa6f2747a171668e46989ae9a26af2d31]

**Review by vladaskorohodova — APPROVED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by ksercs — APPROVED** [review]

**Pull request merged**